src/Hng.Application/Features/Notifications/Dtos/UpdateNotificationDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace Hng.Application.Features.Notifications.Dtos
+{
+    public class UpdateNotificationDto
+    {
+        public bool IsRead { get; set; }
```

Comment:
Should this be snake case, according to nestjs repo

Reply:
corrected